src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2718,26 +1779,18 @@ bool processOnChangeConditionForSubscription
   const std::string&               tenant,
   const std::string&               xauthToken,
   const std::vector<std::string>&  servicePathV,
-  const std::string&               qFilter
+  Restriction*                     resP
```

Comment:
I tried - `const Restriction&` was my first attempt, of course.
But, the compiler didn't let me so I had to use pointers instead ...

NTC